apps/cyberstorm-storybook/public/css/reset.css

```diff
@@ -0,0 +1,175 @@
+/*
+  Made by Elly Loel - https://ellyloel.com/
+  With inspiration from:
+    - Josh W Comeau - https://courses.joshwcomeau.com/css-for-js/treasure-trove/010-global-styles/
+    - Andy Bell - https://piccalil.li/blog/a-modern-css-reset/
+    - Adam Argyle - https://unpkg.com/open-props@1.3.16/normalize.min.css / https://codepen.io/argyleink/pen/KKvRORE
+
+  Notes:
+    - `:where()` is used to lower specificity for easy overriding.
+*/
+
+* {
+  /* Remove default margin on everything */
+  margin: 0;
+  /* Remove default padding on everything */
+  padding: 0;
+  /* Calc `em` based line height, bigger line height for smaller font size and smaller line height for bigger font size: https://kittygiraudel.com/2020/05/18/using-calc-to-figure-out-optimal-line-height/ */
+  line-height: calc(0.25rem + 1em + 0.25rem);
```

Comment:
We can always override this if needed 😈. I think this should be ok. How would you check if it's gonna work? Ofc it can be removed if it is something we do not want to "reset" here.

Reply:
What I meant was that this might work for us as is in the sense that it might give us sensible line heights regardless of font-size, and while not being the exact same ones as in the Figma file, they might be close and be pleasing to ones eye. If so, we can go with it, or if not, we can override it.

Reply:
LGTM for now, shouldn't cause any issues. If any problems arise, it is easy to fix.

Reply:
Well, it's easy to fix now. It might not be as easy to fix once a hundred components have left the line-height undefined and depends on this line. Just saying 🤷🏻‍♂️

Reply:
Yeah, that's true. No reason to have it, I removed it now.

Reply:
TBF independent components should have them defined anyway to avoid the rules trickling down from elsewhere. For example, a button probably should have a line-height of 1, since the white-space is handled with padding and/or w/h declarations.

The general line-height, which could make use this formula - or not - would apply for the text content, ie. headings and body. What this particular line tries to achieve is to reduce individually declared line-heights for different sized text objects to a single, relative formula.